rust/gui-client/src-tauri/deb_files/firezone-client-ipc.service

```diff
@@ -37,7 +37,6 @@ SystemCallArchitectures=native
 SystemCallFilter=@aio @basic-io @file-system @io-event @ipc @network-io @signal @system-service
 UMask=077
 
-Environment="FIREZONE_DNS_CONTROL=systemd-resolved"
```

Comment:
Redundant since the GUI + IPC service only support `systemd-resolved` now. If anyone needs `/etc/resolv.conf` for the GUI we can do that but it probably isn't worth supporting